CodeEdit/Features/CEWorkspace/Models/CEWorkspaceFileManager.swift

```diff
@@ -220,7 +222,12 @@ final class CEWorkspaceFileManager {
                     // TODO: Handle workspace root changing.
                     continue
                 case .itemCreated, .itemCloned, .itemRemoved, .itemRenamed:
-                    try? self.rebuildFiles(fromItem: parentItem)
+                    do {
+                        try self.rebuildFiles(fromItem: parentItem)
+                    } catch {
+                        // swiftlint:disable:next line_length
+                        self.logger.error("Failed to rebuild files for event: \(event.eventType.rawValue), path: \(event.path, privacy: .sensitive)")
```

Comment:
```suggestion
                        self.logger.error(
                            "Failed to rebuild files for event: \(event.eventType.rawValue), path: \(event.path, privacy: .sensitive)"
                            )
```

Reply:
The line is still too long even with that. The other limitation is that OSLog requires log messages to be static strings, so we also can't do something like
```swift
self.logger.error(
     "Failed to rebuild files for event: \(event.eventType.rawValue)"
      + ", path: \(event.path, privacy: .sensitive)"
)
```